### PR TITLE
docs(pipeline-expressions): Minor editing for clarity and format, part 1

### PR DESCRIPTION
**Pull request by dorbin**

### CODE REVIEW

**Comment by dorbin** [comment]
@skim1420 , PTAL

File: guides/user/pipeline-expressions/index.md [hunk 1]
```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
...
 
-![](3.png)
+> **Warning**
+>
+> Sometimes adding an expression stops your ability to use the UI. For example, if you use a parameter for the account field in a cluster deployment Stage, you will see a spinning loader from the UI when you try to edit this cluster.
 
 You can usually type in a value in the text fields of the Pipeline stages in Spinnaker. 
 
+[comment]: <> ![](3.png)
+<img width="600" src="3.png" alt="alt_text">
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md]: Should this be `alt_text`? or something more descriptive?

File: guides/user/pipeline-expressions/index.md [hunk 2]
```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
 
 ![](2.png)
 
-When the pipeline runs, it will substitute the value I have entered for `command` with the value `sleep 40000` that I have entered. 
+When the pipeline runs, it resolves the expression to what I’ve entered for `command`.
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md:27]: Everything I've written has been in 2nd person so far, but I see the rest of this guide is in 1st person. Which one should we pick for the sake of consistency?

File: guides/user/pipeline-expressions/index.md [hunk 3]
```diff
@@ -80,6 +71,18 @@ You can add expressions to be checked here by selecting *Expression* from the Ch
 
 When the pipeline runs, if this expression evaluates to true, the pipeline will continue as is. If not, the stages following this stage will not run. 
 
+### Sometimes you can't use an expression
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md:74]: I'd add a newline after this header. Some markdown rendering requires it.

File: guides/user/pipeline-expressions/index.md [hunk 4]
```diff
@@ -80,6 +71,18 @@ You can add expressions to be checked here by selecting *Expression* from the Ch
...
+There are often cases where the value you want to replace doesn't allow typing, such as specifying a list of security groups or wanting to dynamically control the behavior of a checkbox. For these cases, you should edit the pipeline JSON directly.
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md]: I guess some of this is in 2nd person as well.

File: guides/user/pipeline-expressions/index.md [hunk 5]
```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
...
 
-When the pipeline runs, it will substitute the value I have entered for `command` with the value `sleep 40000` that I have entered. 
+When the pipeline runs, it resolves the expression to what I’ve entered for `command`.
 
 ## Where can I use Pipeline Expressions in Spinnaker?
 
-![](3.png)
+> **Warning**
+>
+> Sometimes adding an expression stops your ability to use the UI. For example, if you use a parameter for the account field in a cluster deployment Stage, you will see a spinning loader from the UI when you try to edit this cluster.
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md]: Why is only "Stage" capitalized?

File: guides/user/pipeline-expressions/index.md [hunk 6]
```diff
@@ -24,33 +24,24 @@ I can now set up my pipeline to take a `command` parameter, when I manually exec
...
-![](4.png)
-
-In the popup screen, you can now edit the fields that match the fields you want to substitute. 
-
-![](5.png)
-
-> **Warning**
->
-> Sometimes adding an expression will stop your ability to use the UI. For example, if you use a parameter for the account field in a cluster deployment Stage, you will see a spinning loader from the UI when you try to edit this cluster.
+> Because expressions are evaluated per stage before each stage starts, you cannot use expressions in the configuration at the pipeline level. For example, expression in the job name of a trigger for the pipeline does not work because no expressions have been evaluated yet.
```
Comment (lwander) [inline guides/user/pipeline-expressions/index.md]: "For example, expression in the job name..." is missing a word I think.

**Review by lwander — APPROVED** [review]
Looks good, just a few questions/comments

File: guides/user/pipeline-expressions/index.md (hunk 2 above)
Comment (dorbin) [reply guides/user/pipeline-expressions/index.md:27]: For *person*, I've left this unchanged for now. Where the original writer used first person I kept it that way in this pass. But you make a good point&mdash;it should be consistent throughout, and I'll make it so. But I'll wait till the next pass on this page to updated it here.

File: guides/user/pipeline-expressions/index.md (hunk 5 above)
Comment (dorbin) [reply guides/user/pipeline-expressions/index.md]: Good question; will change.

File: guides/user/pipeline-expressions/index.md (hunk 1 above)
Comment (dorbin) [reply guides/user/pipeline-expressions/index.md]: Something more descriptive ;) 
I put it there to test the image b/c I was having trouble with the markdown syntax for the image.

File: guides/user/pipeline-expressions/index.md (hunk 3 above)
Comment (dorbin) [reply guides/user/pipeline-expressions/index.md:74]: Will do

**Pull request merged**